objects/src/transaction/executed_tx.rs

```diff
@@ -27,6 +27,7 @@ pub struct ExecutedTransaction {
     account_delta: AccountDelta,
     tx_args: TransactionArgs,
     advice_witness: AdviceInputs,
+    tx_progress: TransactionProgress,
```

Comment:
In case some of the fields in the `TransactionProgress` are `None`s, the corresponding values in the `TransactionMeasurement` should be 0?

Reply:
Yes - I think that's fine.

Reply:
We will probably remove `serde` soon from `miden-objects`, `miden-lib`, and `miden-tx` crates (we don't really need it there any more). So, if we want to use `serde`, it should be located in `bench-tx`. So, basically, we need to have the following:

- `TransactionProgress` in `miden-tx` crate - as it was.
- `TransactionMeasurements` in `miden-objects` crate. The main reason for this is that `TransactionProgress` is a struct specific to tracking progress of the transaction as it is being executed. It has quite a few things which we don't really need in `ExecutedTransaction` - so, having a dedicated `TransactionMeasurements` struct makes sense, IMO.
- If we can't use `TransactionMeasurements` directly in `bench-tx` (e.g.., because we can't implement `serde` traits on it), I'd still have `TransactionBenchmark` and convert `TransactionMeasurements` into it.